tests/php/Controller/ExAppsPageControllerTest.php

```
@@ -169,4 +170,55 @@ public function testListAppsPicksTranslationForInjectedLanguage(): void {
+		$this->config->expects(self::once())
+			->method('setSystemValue')
+			->with('app_install_overwrite', ['other_app', 'my_exapp']);
+
+		self::assertInstanceOf(JSONResponse::class, $this->controller->force('My_ExApp!'));
+	}
+
+	public function testForceDoesNotRewriteWhenAlreadyMarked(): void {
+		$this->appManager->method('cleanAppId')->willReturn('my_exapp');
+
+		$this->config->method('getSystemValue')
+			->with('app_install_overwrite', self::anything())
```

PHPStan (2.2.6)

[error] Call to an undefined static method OCA\AppAPI\Tests\php\Controller\ExAppsPageControllerTest::anything().


(staticMethod.notFound)